core-services/prow/02_config/_plugins.yaml

```diff
@@ -1207,6 +1207,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-2.x-configs
+    ci-operator/config/stackrox/stackrox/*-release-4.0*.yaml:
+      clusters:
+        app.ci:
+        - ci
+      gzip: true
+      name: ci-operator-4.0-configs
```

Comment:
I had to use this because of CI failure https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/37753/pull-ci-openshift-release-master-correctly-sharded-config/1641150492815921152

Please see my commit message where I tried to explain this and other things [`513b6fd` (#37753)](https://github.com/openshift/release/pull/37753/commits/513b6fd858cebdaad6a90afa6ca6c63308119796)